api/replication.storage/v1alpha1/volumereplication_types.go

```diff
@@ -36,6 +36,25 @@ const (
 	ConditionValidated = "Validated"
 )
 
+// These are valid messages for various
+// conditions and states of Volume
+// replication.
+const (
+	MessagePromoted               = "volume is promoted"
+	MessageHealthy                = "volume is healthy"
```

Comment:
This is not verbose enough, not clear what is "healthy".

Reply:
this just states that volume is in healthy state, not sure what other information you want to be printed here. @Madhu-1 any suggestions from your end?